x-pack/plugins/fleet/common/services/agent_policies_helpers.ts

```diff
@@ -13,6 +13,12 @@ import {
   FLEET_ENDPOINT_PACKAGE,
 } from '../constants';
 
+export function getDefaultFleetServerpolicyId(spaceId?: string) {
+  return !spaceId || spaceId === '' || spaceId === 'default'
```

Comment:
is it possible to compare using `DEFAULT_NAMESPACE_STRING` from `@kbn/core-saved-objects-utils-server`? (not sure if that can be imported into common)